Source/Scene/Cesium3DTileset.js

```diff
@@ -351,6 +354,18 @@ define([
         this.debugShowViewerRequestVolume = defaultValue(options.debugShowViewerRequestVolume, false);
 
         /**
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, draws labels to indicate the geometric error of each tile.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.debugShowGeometricError = defaultValue(options.debugShowGeometricError, false);
+        this._geometricErrorLabels = new LabelCollection();
```

Comment:
Initialize this to `undefined` so users don't pay for features they don't use.  Likewise, when `debugShowGeometricError` is set to `false` destroy this object.